aws-cloud-integrations.md

```diff
@@ -328,15 +328,17 @@ Download template files from the URLs provided below and upload them as the stac
 ### Step 4: Attaching IAM permissions to Kubecost
 Now that the policies have been created, attach those policies to Kubecost. We support the following methods:
 
+> **NOTE:** If you are using the alternative [multi-cloud integration](https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/multi-cloud) method steps 4 and 5 are not required. The use of "Attach via Pod Annotation on EKS" authentication is optional.
```

Comment:
Small detail. A comma between "method" and "steps" may increase readability?

Reply:
Thanks, I will add them.